### PR TITLE
Update changelog for December 6th release

### PR DESCRIPTION
**Pull request by VincentDondain**
Content of changelog taken from analysing over 92 commits and [selecting key changes](https://docs.google.com/document/d/1qOihyrT_DlP56cbhNu1f_EGv6PEgidSGGH1QOyhDsuc/edit?tab=t.0), excluding internal only ones.

### CODE REVIEW

File: changes.md [hunk 1]
```diff
@@ -1,3 +1,52 @@
+## 📅 6 December 2024
+
+**Note:** Future updates to the changelog will be available at [github.blog/changelog](https://github.blog/changelog).
+
+### Image Preview Support
+
+You can now preview selected images directly from the file tree in Workspace.
```
Comment (chrispat) [inline changes.md:7]: The wording here sounds like you can only preview certain images or maybe certain image types. Is there anything new you can do with the images? Can you create image assets in workspace, if so maybe the value is you can now review images in your workspace. Perhaps we link this to continued improvements from the previous changelog https://github.com/githubnext/copilot-workspace-user-manual/blob/28e07942a6392eb7164275cca0e20bb4c4afa6ac/changes.md#file-attachments

Comment (VincentDondain) [reply changes.md:7]: I don't think we can link this to file attachement, this is really about seeing the asset in the editor

Comment (VincentDondain) [reply changes.md:7]: How about: "You can now preview any image file you select from the file tree in Workspace."

File: changes.md [hunk 2]
```diff
@@ -1,3 +1,53 @@
+## 📅 6 December 2024
+
+As you may have seen in Discord a few weeks ago, [Copilot Workspace is graduating](https://discord.com/channels/735557230698692749/1309719558449397780/1309719558449397780)! It is a very exciting time, and also a time of change. So before getting into the product changes from this week, we want to highlight a few logistical changes, because everyone loves logistics :muscle:
+
+**Changelog location:** Starting next week we will post changelogs to the main [GitHub changelog](https://github.blog/changelog) rather than this repository. This will be the last changelog posted in this repository, so make sure to follow the [GitHub changelog](https://github.blog/changelog) to stay up to date with the latest and greatest.
+
+**How to provide feedback:** We are also transitioning from the current Discord to a [GitHub Discussion](https://gh.io/workspace-feedback) as the primary place for feedback and discussions around Copilot Workspace. We will still be available in Discord, but posting in the [discussion](https://gh.io/workspace-feedback) will ensure we see your feedback.
```
Comment (entcheva) [inline changes.md:7]:
```suggestion
**How to provide feedback:** We are also transitioning from the current Discord to a [GitHub Discussion](https://gh.io/workspace-feedback) as the primary place for feedback and discussions around Copilot Workspace. We will still be available in Discord, but posting in the [discussion](https://gh.io/workspace-feedback) will ensure we see your feedback sooner.
```

File: changes.md [hunk 3]
```diff
@@ -1,3 +1,53 @@
...
+
+Okay, no more logistics, now onto the product changes for this week! :tada:
```
Comment (entcheva) [inline changes.md:9]:
```suggestion
Okay, now onto the product changes for this week! :tada:
```

File: changes.md [hunk 4]
```diff
@@ -1,3 +1,53 @@
...
+
+**After:**  
+<img src="https://github.com/user-attachments/assets/2f5a89f6-00c7-42cc-81aa-381f928cb34c" width="400" >
+
+### VS Code Extension Updates
+
+- **Stale View Fix:** Resolved an issue where stale view states were retained in certain views.
+- **Push to Branch / PR Creation Fix:** Fixed failures when merging into an existing branch with updates to the same files.
+- **Binary Detection Fix:** Addressed a false positive issue where folders were incorrectly flagged as binary after session syncing stopped.
+- **Enhanced Session List:** Sessions now appear earlier in their lifecycle in the session list, supporting the new brainstorming feature in VS Code.
```
Comment (entcheva) [inline changes.md:48]: Should we pull this up above the bug fixes?

**Review by entcheva — APPROVED** [review]
Added a few comments, not blocking. Thank you!

File: changes.md (hunk 4 above)
Comment (cmuto09) [reply changes.md:48]: Yeah that's a good call

**Review by skylar-anderson — APPROVED** [review]

**Pull request merged**